addons/account_lock/__openerp__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
```

Comment:
the file should not be called **manifest**?

Reply:
not in 9.0 afair :-)

Reply:
iups!